### PR TITLE
[feat] added week 5 report

### PR DESCRIPTION
**Pull request by achintya-7**
## Project
DIGIT - Vehicle Tracker

### Changes
Added Week 5 Repor

### CODE REVIEW

File: docs/2023/DIGIT/Vehicle Tracker/updates/2023-08-04.md [hunk 1]
```diff
@@ -4,13 +4,21 @@ author: Achintya
...
+- [x] Added I18N via API and stored it in local storage
+- [x] Added MDMS via API and stored it in local storage
+- [x] Completed Home and Detail Info Screen
 
 ## Screenshots / Videos 
+There were some changes in the figma design so I have remade/modified some of my previous screens.
+
+| Home | Detail |
+| --- | --- |
+| ![](../static/week5/img-1.jpg) | ![](../static/week5/img-2.jpg) |
```
Comment (photon0205) [inline docs/2023/DIGIT/Vehicle Tracker/updates/2023-08-04.md:16]: just give a sensible alt names to the images, rest is good

**Review by photon0205 — CHANGES_REQUESTED** [review]

File: docs/2023/DIGIT/Vehicle Tracker/updates/2023-08-04.md (hunk 1 above)
Comment (achintya-7) [reply docs/2023/DIGIT/Vehicle Tracker/updates/2023-08-04.md:16]: Ok

**Comment by achintya-7** [comment]
@photon0205 can you re review it

File: docs/2023/DIGIT/Vehicle Tracker/updates/2023-08-04.md [hunk 2]
```diff
@@ -4,13 +4,21 @@ author: Achintya
...
 
 ## Screenshots / Videos 
+There were some changes in the figma design so I have remade/modified some of my previous screens.
+
+| Home | Detail |
+| --- | --- |
+| ![new home screen](../static/week5/img-1.jpg) | ![new detail screen](../static/week5/img-2.jpg) |
 
 ## Contributions
+- [Init Setup for API calls](https://github.com/egovernments/egov-rnd/commit/44119722cd9e45315094d5eb8647b57d994e91e6)
```
Comment (shubhigupta991) [inline docs/2023/DIGIT/Vehicle Tracker/updates/2023-08-04.md:19]: Please add PR links from now onwards.

**Review by shubhigupta991 — CHANGES_REQUESTED** [review]

File: docs/2023/DIGIT/Vehicle Tracker/updates/2023-08-04.md (hunk 2 above)
Comment (achintya-7) [reply docs/2023/DIGIT/Vehicle Tracker/updates/2023-08-04.md]: done

**Pull request merged**